src/validation.js

```diff
@@ -0,0 +1,75 @@
+function validateOptions ({ inline, compilers, paths, languages, cache, variables, aliases }) {
+  return [
+    ...arePathsValid(paths),
+    ...isInlineValid(inline),
+    ...areCompilersValid(compilers),
+    ...areLanguagesValid(languages),
+    ...isCacheValid(cache),
+    ...areAliasesValid(aliases)
+  ]
+}
+
+function arePathsValid (paths) {
+  if (!Array.isArray(paths)) return ['Compiler option "paths" must be an array']
+  if (paths.some(path => typeof path !== 'string')) {
+    return ['Compiler option "paths" must contain only strings']
+  }
+  if (paths.some(path => path === '')) {
+    return ['Compiler option "paths" cannot contains empty strings']
+  }
+  return []
+}
+
+function isInlineValid (inline) {
+  const SUPPORTED_TYPES = ['images', 'scripts', 'stylesheets']
+  if (!Array.isArray(inline)) return ['Compiler option inline must be an array']
+  if (inline.some(option => !SUPPORTED_TYPES.includes(option))) {
+    return [`Inline option must be one of ["images", "scripts", "stylesheets"]`]
+  }
+  return []
+}
+
+function areCompilersValid (compilers) {
+  if (Object.prototype.toString.call(compilers) !== '[object Object]') return ['Compiler option "compilers" must be an object']
```

Comment:
pure-utilities isObject ?

Reply:
pure-conditions isObject zwróci też true dla funkcji i tablicy.
Można dodać funkcję, która będzie sprawdzać czy jest to obiekt

Reply:
można dodać  `isPlainObject`